src/main/resources/static/templates/learn.html

```diff
@@ -63,6 +63,12 @@ <h1>Curated List of Learning Material</h1>
                                 <p class="desc">Hands-on, targeted at an audience new to reactive programming, with Reactor 3.</p>
                                 <p class="link"><a href="https://tech.io/playgrounds/929/reactive-programming-with-reactor-3">See more <span class="fa fa-angle-right"></span></a></p>
                             </article>
+                            <article>
+                                <p class="title"><span class="fa fa-github">&nbsp;</span>
+                                    Practical Reactor workshop</p>
+                                <p class="desc">Practical hands-on workshop, that includes over 100 unique exercises to provide production like experience and possibility improve existing skills.</p>
```

Comment:
looking at the other entries, contributions that are not from a Reactor team resource are kind of denoted by a "by Name". Can you append your name there @schananas ?

Reply:
Done.